tests/components/cisco_webex/test_sensor.py

```diff
@@ -0,0 +1,37 @@
+"""Test the Cisco Webex config flow."""
+
+from unittest.mock import patch
+
+from homeassistant.components.cisco_webex.sensor import WebexPresenceSensor
+
+from tests.components.cisco_webex.mocks import MockWebexTeamsAPI
+
+MOCK_API = MockWebexTeamsAPI(access_token="123")
+SENSOR = WebexPresenceSensor(api=MOCK_API, email="test@123.com", name="Name")
```

Comment:
Set up the integration and let it add an entity whioe patching the library to avoid I/O. Don't create the entity directly.